### DISCUSSION TITLE
Navigations

### DISCUSSION THREAD

**Discussion by AaronSadlerUK** [TimGeyssens/Umbraco-v8-Community-Website-StarterKit#54] [Ideas]
So to make managing the menu a breeze out of the box, I propose that https://github.com/AaronSadlerUK/Umbraco.MegaNavV8 is incorporated with the package.

I'm not sure if there are any alternatives, so please do add those in the comments.

**Comment by TimGeyssens** [comment]
Yeah never used it but looks good :) mostly I just use multi url picker but then you don't the levels... so this one feels better :) I'll add an issue...

**Comment by TimGeyssens** [comment]
does it work with models builder?

**Reply by AaronSadlerUK** [reply]
Yep :)

Well it did last time I tested it with models builder in DLL mode.

**Reply by TimGeyssens** [reply]
sweet! then I would say go ahead :)
